api/controllers/height_controller.py

```
@@ -0,0 +1,284 @@
+        try:
+            if not self.validate_latest_entry(height):
+                raise ValueError("height or latest entry not valid")
+
+            self.connect()
+            cursor = self.conn.cursor()
+            cursor.execute(SAVE_END_HEIGHT_QUERY.format(height))
+            self.conn.commit()
+            cursor.execute(SAVE_START_HEIGHT_QUERY.format(height))
+            self.conn.commit()
+        except Exception as e:
+            raise RuntimeError(e)
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)